src/components/PracticeSection/MultipleChoice.js

```diff
@@ -542,11 +542,8 @@ export default function MultipleChoice() {
                               <Form.Check
                                 type="radio"
                                 value={option}
-                                label={
-                                  <div
-                                    dangerouslySetInnerHTML={{ __html: option }}
```

Comment:
I never remember: do we know why this was used? 
Could it be that it was used for displaying more complicated answers (e.g. fractions?)

Reply:
dangerouslySetInnerHTML is a way to get react to render HTML tags from a rich text editor.

I do not know why this is used to render the value for the label. 
From what I've read, removing the 'dangerouslySetInnerHTML' will not change the functionality in any way.

Reply:
@anneschjoedt @ffthe  - was there a reason for using `dangerouslySetInnerHTML`? do you still remember?